.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,83 @@
+name: build-pr
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
+jobs:
+  build-pr:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get Composer Cache Directory
+      id: composer-cache
+      run: |
+        echo "::set-output name=dir::$(composer config cache-files-dir)"
+    - uses: actions/cache@v1
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-composer-
+    - name: Composer Install
+      uses: php-actions/composer@v1
+      with:
+        args: install
+    - name: Read .nvmrc
```

Comment:
That's a nice touch. I usually just specify the node version in the action, but I like that! 👍